lib/ex_trello/api/boards.ex

```diff
@@ -49,7 +49,6 @@ defmodule ExTrello.API.Boards do
 
   # Board labels
   def labels(board), do: labels(board, [])
-  def labels(%Board{id: id}, options), do: labels(id, options)
```

Comment:
Add the line back in and add 1 more test demonstrating its usage (which will satisfy the missing coverage)

e.g.

```elixir
{:ok, board} = ExTrello.board("57fb9dfd566ee0dbedbc100b")
# ...
# imagine something is being done here
# ...
{:ok, labels} = ExTrello.board_labels(board, limit: 2)
```

After that's done I'll be happy to merge it in.